routes/web.php

```diff
@@ -1,5 +1,10 @@
 <?php
 
+use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Facades\Auth;
+use App\Http\Controllers\Auth\ChangePasswordController;
+use App\Http\Controllers\{HomeController, UsersController, BirthdayController, CouplesController, FamilyActionsController, UserMarriagesController};
```

Comment:
This line looks too long, I like this way better.

```suggestion
use App\Http\Controllers\Auth\ChangePasswordController;
use App\Http\Controllers\BackupsController;
use App\Http\Controllers\BirthdayController;
use App\Http\Controllers\CouplesController;
use App\Http\Controllers\FamilyActionsController;
use App\Http\Controllers\HomeController;
use App\Http\Controllers\UserMarriagesController;
use App\Http\Controllers\UsersController;
use Illuminate\Support\Facades\Auth;
use Illuminate\Support\Facades\Route;
```

Reply:
done